docs/jssandbox_tutorial/index.rst

```diff
@@ -0,0 +1,58 @@
+Javascript Sandbox Tutorial!
+===========================================================
+
+This is a javascript sandbox tutorial for writing Vumi Go Application.
```

Comment:
Some of these sections are going to be in their own files. I would focus on just the four initial sections in this file and leave the others for later: Four sections:

* What is the sandbox?
* Introduction of the example we're going to use?
* Outcomes of the tutorial
* Other documentation